addons/bus/models/res_partner.py

```diff
@@ -9,12 +9,14 @@ class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     im_status = fields.Char('IM Status', compute='_compute_im_status')
+    last_seen = fields.Datetime('User last Seen', compute='_compute_im_status')
```

Comment:
@tde-banana-odoo just to make sure, do you confirm that it is reasonable to add this computed field?